.github/workflows/build-package.yml

```diff
@@ -196,7 +196,7 @@ jobs:
 
       - uses: ./.github/setup-kind
         with:
-          traefik-chart-version: 27.0.2
+          traefik-chart-version: 31.1.1
```

Comment:
Isn't it safer to still use chart version 27.0.2, just like we do in our production cluster?
If we upgrade traefik to v3 in prod, we can also bump it here.

Reply:
I checked it's a minor Traefik update, I want to propose to actually update in scaleway to this version as well.

Reply:
Moved it back to `27.0.2`, everything still passes. The good news is now we know these changes work on both Traefik versions 😉